README.md

```diff
@@ -108,9 +108,10 @@ On iOS it is possible to configure the accessibility of the keychain by setting
  * AfterFirstUnlockThisDeviceOnly
  * WhenUnlocked
  * WhenUnlockedThisDeviceOnly
+ * WhenPasscodeSetThisDeviceOnly (this option is available only on iOS8 and later)
+but NOT to one of the following:
  * Always
  * AlwaysThisDeviceOnly
```

Comment:
I don't think you should list them then.
If you want to warn others who might attempt to use them,  put it in a following paragraph.
Not in the list of options.

It reads very confusingly in the rich text.